src/app/content/components/Toolbar/index.tsx

```diff
@@ -116,9 +118,12 @@ class Toolbar extends React.Component<Props, State> {
             }
           </Styled.SearchInputWrapper>
         </Styled.SearchPrintWrapper>
-        <StudyGuidesButton />
-        <HighlightButton/>
+        <Styled.NudgeElementTarget id={nudgeStudyToolsTargetId}>
+          <StudyGuidesButton />
+          <HighlightButton />
+        </Styled.NudgeElementTarget>
         <PrintButton />
+        <NudgeStudyTools />
```

Comment:
Hmm, not sure how it could work, because we have to create `spotlight` div and `background` div and `spotlight` div has to be positioned above the `background` and directly above the elements to highlight